### ISSUE TITLE
ensure commands in docs and readme also work in windows

### ISSUE THREAD

**Issue by ankur22** [grafana/xk6-browser#592]

**Comment by imiric** [comment]
Thinking about this, would it make sense for xk6 to automatically append the extension on Windows? After all, it shouldn't produce unusable binaries, and we shouldn't expect users to have to keep this in mind. cc: @mstoykov

**Comment by imiric** [comment]
Right. I think it should, though, but only if it's not already specified. After all, it should never produce unusable binaries, and files without `.exe` on Windows are certainly that, and need to be renamed manually to work. It would probably be easier to do that automatically and save us the documentation and support work around it.

**Comment by inancgumus** [comment]
@mdcruz, I believe you've already [fixed](https://k6.io/docs/javascript-api/xk6-browser/get-started/installation/) this issue too 😅 (that's why I assigned you again, so you can close it!)

**Comment by mdcruz** [comment]
Closing this issue and going forward, we will link the docs installation guide from the README instead to avoid any duplication
